src/navigation/navigation.scss

```diff
@@ -55,6 +55,14 @@ nav {
                font-size: 14px;
                cursor: pointer;
             }
+            .subgroup {
+               min-width: 130px;
```

Comment:
Use rem please.

Reply:
@hirad-rewok since base font-size in webtrader is `14px`, that would be difficult to rely on `rem`. 
I suggest keeping it as `px`

Reply:
Yes, we don't really use `rem` in this project and since it's not set I want to avoid 
![situations like this](https://user-images.githubusercontent.com/71861519/195751189-633a6eea-d09d-4b34-b3bb-f47b88f32579.png) inside the codebase